opal/corelib/kernel.rb

```diff
@@ -99,6 +100,10 @@ def Array(object)
     }
   end
 
+  def __dir__
+    File.realpath(File.dirname(__FILE__))
```

Comment:
__FILE__ here is opal/corelib/kernel.rb, so __dir__ is always opal/corelib

Reply:
This __dir__ ___FILE___ thing probably needs some further discussion/work. I add it to the TODO list above.